wcc-contentful-app/lib/generators/wcc/templates/menu/generated_add_menus.ts

```diff
@@ -174,19 +264,124 @@ export = function (migration: Migration) {
     validations: [],
     disabled: false,
     omitted: false,
-    items:
+    items: {
+      type: 'Link',
+      validations: [
+        {
+          linkContentType: ['dynamicButton', 'menuButton']
+        }
+      ],
+      linkType: 'Entry'
+    }
+  });
+
+  dropdownmenu.changeEditorInterface('name', 'singleLine', {
+    helpText:
+      "If you don't set a menu label, this is the text that will appear on the button that opens the dropdown menu.  If you do set a menu label, that will control the text."
+  });
+  dropdownmenu.changeEditorInterface('label', 'entryLinkEditor');
+  dropdownmenu.changeEditorInterface('items', 'entryLinksEditor');
+  dropdownmenu.changeEditorInterface('internalTitle', 'singleLine');
+
+  var menuitem = migration.createContentType('menuItem', {
```

Comment:
the whole `menuItem` content type is specific to ThePorch and is not included in the menu system that we use for Watermark Resources.  In fact we're trying to get rid of `menuItem` on ThePorch because it's not flexible enough.

Reply:
I remember you mentioning this the other day, but decided to include it anyway since it seemed to be specifically named as a requirement in the zube ticket: https://zube.io/watermarkchurch/development/c/3295
But yes, I understand more clearly now and will go ahead and take it out.